### ISSUE TITLE
Would nn.BatchNorm2d work as expected?

### ISSUE THREAD

**Issue by lime-j** [google/torchax#22]
Dear authors,

Thanks for your wonderful work! I wonder if nn.BatchNorm2d works. During training, the running mean and running variance are updated according to their own rule instead of being updated with the optimizer.
